### DISCUSSION TITLE
SKUs on Shop Receipt Transactions changing

### DISCUSSION THREAD

**Discussion by campodea** [etsy/open-api#1112] [Q&A]
It seems that if a SKU is changed on a listing that even for existing orders, the SKU that is reported back on Shop Receipt Transactions is also changed. Even though Etsy UI will remain showing the original SKU that was on the item when the order was originally placed.

Is there a way in this scenario to get the original SKU for a transaction instead of the updated one?

**Comment by campodea** [comment]
I am pretty certain this did not used to be the case, at least not in V2

**Comment by emergingdzns** [comment]
This sounds like a bug. I would highly recommend emailing developers@etsy.com and give them this info. Ideally also provide a listing id, transaction id as well as the output of the API call showing the updated sku and a screenshot of the Etsy UI showing the correct one.

**Reply by campodea** [reply]
Thank you for the reply. I had emailed them but I will follow up with screenshots as well!
